registration/templates/registration/admin_approve_email.html

```diff
@@ -0,0 +1,35 @@
+{% load i18n %}
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>{{ site.name }} {% trans "registration" %}</title>
+</head>
+
+<body>
+<p>
+    {% blocktrans with site_name=site.name %}
+    The following user (NEED TO ADD USER HERE) has asked to register an account at
+    {{ site_name }}.
+    {% endblocktrans %}
+</p>
+<p>
+    {% blocktrans %}
+    To approve this, please click the following link.
```

Comment:
why not just add the `a href` in this paragraph? So it would read something like `To approve this, please <a href=[...]>click here</a>`